MM2RandoLib/Randomizers/RWeaknesses.cs

```diff
@@ -72,32 +260,46 @@ public void Randomize(Patch in_Patch, RandomizationContext in_Context)
         /// </summary>
         private void RandomizeU(Patch in_Patch, ISeed in_Seed)
         {
-            List<EDmgVsBoss> bossPrimaryWeaknessAddresses = EDmgVsBoss.GetTables(false, true);
-            List<EDmgVsBoss> bossWeaknessShuffled = new List<EDmgVsBoss>(bossPrimaryWeaknessAddresses);
-            bossWeaknessShuffled = in_Seed.Shuffle(bossWeaknessShuffled).ToList();
+            Dictionary<EWeaponIndex, EDmgVsBoss> bossPrimaryWeaknessAddresses = EDmgVsBoss.GetTables(false, true);
```

Comment:
Could we add explicit parameter names in the calls when passing in booleans like this? It would help the reader understand what is happening.

Reply:
I didn't write the original call, but yeah I can add them in. I tend to think passing raw bools is bad design. I wish enums were strongly typed. Really missing algebraic data types the more I use C#.

Anyway, I changed it.